### PR TITLE
Small bug fixes

### PR DESCRIPTION
**Pull request by Kurdran**
Found a redneck fix for archaeological biome recognition (when calculating the reputation points stopgap)
 Added new options for messing about with season lenghts 
 Fixed a small issue with the impatience multiplier description in the readme file
 Removed some log spamming lines

### CODE REVIEW

File: Plugin.cs [hunk 1]
```diff
@@ -122,11 +122,21 @@ public static void AddReputationPoints_PrePatch(ReputationService __instance, fl
...
+            WriteLog(biomeName);
             var newAmount = amount * cheatConfig.ReputationMutiplier;
+            // it'd be nice if we could figure out when an archaeology dig site is available based off the buildings list
+            //var maxReputation = Serviceable.BuildingsService.Relics.Count == 0
+            //    ? (float)__instance.GetReputationToWin() - cheatConfig.ReputationStopgap
+            //    : (float)__instance.GetReputationToWin() - cheatConfig.ReputationStopgap - 1;
             // rep stopgaps should change a bit based on events in the map, such as archaeologist ruins
-            var maxReputation = Serviceable.BuildingsService.Altars.Count == 0
+            // no I don't like hardcoded strings, but it'll have to do for now...
+            var maxReputation = Serviceable.BiomeService.CurrentBiome.Name != "Scarlet Orchard"
```
Comment (Josiwe) [inline Plugin.cs]: Best practice would be to use ToUpperInvariant and compare it to "SCARLET ORCHARD"

File: Plugin.cs [hunk 2]
```diff
@@ -122,11 +122,21 @@ public static void AddReputationPoints_PrePatch(ReputationService __instance, fl
             if (!__instance.IsValidReputationGain(amount) || cheatConfig == null || Serviceable.BuildingsService.Seals.Count > 0)
                 return;
 
+            var biomeName = Serviceable.BiomeService.CurrentBiome.Name;
+            WriteLog(biomeName);
+            WriteLog(biomeName);
+            WriteLog(biomeName);
```
Comment (Josiwe) [inline Plugin.cs]: Do we need to write this three times?

File: Plugin.cs [hunk 3]
```diff
@@ -175,6 +185,12 @@ public static bool GetRerollsLeft_PrePatch()
             if (cheatConfig == null || !cheatConfig.EnableInfiniteCornerstoneRerolls)
                 return true; // run the original game method
 
+            var biomeName = ;
+            WriteLog(biomeName);
+            WriteLog(biomeName);
+            WriteLog(biomeName);
```
Comment (Josiwe) [inline Plugin.cs]: This seems superfluous

**Review by Josiwe — CHANGES_REQUESTED** [review]
Just a few notes. If you can fix them up I'll approve for merge.

File: Plugin.cs (hunk 2 above)
Comment (Kurdran) [reply Plugin.cs]: That was unintended lol

File: Plugin.cs (hunk 3 above)
Comment (Kurdran) [reply Plugin.cs]: same as above

File: Plugin.cs (hunk 1 above)
Comment (Kurdran) [reply Plugin.cs]: Good catch, thx

File: Plugin.cs [hunk 4]
```diff
@@ -82,6 +82,49 @@ public static void Setup_PostPatch(CameraController __instance)
...
         #endregion
 
+        #region GetSeasonLenghtRate_PrePatch
+        /// <summary>
+        /// Modify the length of each season
+        /// </summary>
+        /// <param name="__instance"></param>
+        /// <param name="season"></param>
+        /// <param name="__result"></param
+        [HarmonyPatch(typeof(EffectsService), nameof(EffectsService.GetSeasonLenghtRate))]
```
Comment (Josiwe) [inline Plugin.cs:92]: I thought this was a typo but they misspelled it themselves lol

**Review by Josiwe — APPROVED** [review]

**Pull request merged**